### PR TITLE
Scheduling an Export: Described configuring export if remote server was selected. Updated entity types

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Described configuring export if remote server was selected. Updated entity types

## Affected documentation pages

https://docs.magento.com/user-guide/system/data-schedule-export.html

whatsnew
Updated the [Scheduling an Export](https://docs.magento.com/user-guide/system/data-schedule-export.html) topic to include additional information about entity types and configuration for remote server.

### CODE REVIEW

File: src/system/data-schedule-export.md [hunk 1]
```diff
@@ -64,6 +71,14 @@ After each export, the export file is placed in the user-defined location, and a
    - For **Local Server**, enter a relative path within the Commerce installation, such as `var/export`. If the Remote storage module is configured, use `import_export/export`.
    - For **Remote FTP server**, enter the full URL and path to the target folder on the destination server.
 
+1. If the **Remote FTP server** was selected enter connection credentials to the server and select additional settings:
```
Comment (leslietilling) [inline src/system/data-schedule-export.md:74]:
```suggestion
1. If the **Remote FTP **  server is selected, enter connection credentials to the server and select additional settings:
```

File: src/system/data-schedule-export.md [hunk 2]
```diff
@@ -64,6 +71,14 @@ After each export, the export file is placed in the user-defined location, and a
...
+
+   - **FTP Host[:Port]**, enter remote ftp host address.
```
Comment (leslietilling) [inline src/system/data-schedule-export.md:76]:
```suggestion
   - For **FTP Host[:Port]**, enter remote FTP host address.
```

File: src/system/data-schedule-export.md [hunk 3]
```diff
@@ -64,6 +71,14 @@ After each export, the export file is placed in the user-defined location, and a
...
+   - **User Name**, enter a username, to access the remote server.
```
Comment (leslietilling) [inline src/system/data-schedule-export.md:77]:
```suggestion
   - For **User Name**, enter the username used to access the remote server.
```

File: src/system/data-schedule-export.md [hunk 4]
```diff
@@ -64,6 +71,14 @@ After each export, the export file is placed in the user-defined location, and a
...
+   - **Password**, enter a password of provided username account.
```
Comment (leslietilling) [inline src/system/data-schedule-export.md:78]:
```suggestion
   - For **Password**, enter the password of the provided username account.
```

File: src/system/data-schedule-export.md [hunk 5]
```diff
@@ -64,6 +71,14 @@ After each export, the export file is placed in the user-defined location, and a
...
+   - **File Mode**, `Binary` or `ASCII`.
```
Comment (leslietilling) [inline src/system/data-schedule-export.md:79]:
```suggestion
   - **File Mode**, `Binary` or `ASCII`.
```
```suggestion
   - For **File Mode**, choose `Binary` or `ASCII`.
```

File: src/system/data-schedule-export.md [hunk 6]
```diff
@@ -64,6 +71,14 @@ After each export, the export file is placed in the user-defined location, and a
...
+   - **Passive Mode**, `No` or `Yes`.
```
Comment (leslietilling) [inline src/system/data-schedule-export.md:80]:
```suggestion
   - For **Passive Mode**, choose `No` or `Yes`.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
minor edits for style and clarity

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**